components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -497,21 +503,25 @@
                                 if (propertyValue != null) {
 
                                     if (isBoolean == 1) { %>
-                            <input type="checkbox" name=<%=value%> id=<%=value%>
+                            <input type="checkbox" name=<%=Encode.forHtmlAttribute(value)%>
+                            id=<%=Encode.forHtmlAttribute(value)%>
                                    class="checkbox" checked/>
                             <%
                             } else if (isBoolean == 0) { %>
-                            <input type="checkbox" name=<%=value%> id=<%=value%>
+                            <input type="checkbox" name=<%=Encode.forHtmlAttribute(value)%>
+                            id=<%=Encode.forHtmlAttribute(value)%>
                                    class="checkbox"/>
                             <%
                             } else if (propertyName.endsWith("password") || propertyName.endsWith("Password")) { %>
-                            <input type="password" name=<%=value%> id=<%=value%> style="width:95%"
-                                   value="<%=propertyValue%>"/>
+                            <input type="password" name=<%=Encode.forHtmlAttribute(value)%>
+                            id=<%=Encode.forHtmlAttribute(value)%> style="width:95%"
```

Comment:
Even here the variable value is hardcoded and does not contain special characters. No need to encode that variable